README.md

```diff
@@ -10,6 +10,40 @@ These files are maintained by a Continuous Integration and Continuous Deployment
 
 ## Contributing
 
+### How do I set up my developerment environment?
```

Comment:
```suggestion
### How do I set up my developer environment?
```

Reply:
Looks good, thanks for the contribution, @GaryGapinski. I will work with Greg to fix this.